### PR TITLE
UNI-374 Fix param finder

### PR DESCRIPTION
**Pull request by marionleborgne**
cc @lscheinkman

### CODE REVIEW

**Comment by marionleborgne** [comment]
@lscheinkman  note:  `npm run install:backend` for my changes to take effect.

**Comment by lscheinkman** [comment]
@marionleborgne It works for me. 
:+1: LGTM

File: unicorn/py/unicorn_backend/debug/nab_params.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{
```
Comment (vitaly-krugl) [inline unicorn/py/unicorn_backend/debug/nab_params.json]: @marionleborgne, are you planning to remove this nab_params.json file?

Comment (marionleborgne) [inline unicorn/py/unicorn_backend/debug/nab_params.json]: @vitaly-krugl The FIXME has been deleted in the pending PR (UNI-357). It was a very short lived fix so I didn't think it needed a JIRA. But it's noted and I will make sure to create a corresponding JIRA for future FIXMEs.

**Pull request merged**